site/_helpers/markdownItContainer.js

```diff
@@ -0,0 +1,167 @@
+/**
```

Comment:
Please move this file into a directory `third-party/markdown-it-container` and add a license file into the directory.

Reply:
Done.